onboardProject/pages/LoginPage.tsx

```diff
@@ -2,24 +2,63 @@ import React from 'react';
 import PrimaryButton from '../atomic/atm/atm.button/button.component';
 import {H1} from '../atomic/atm/atm.typo/typo.style'
 import { TextField } from '../atomic/atm/atm.input/input.component'
-import { View } from 'react-native';
+import { View, Alert } from 'react-native';
+import PageContainer from '../atomic/atm/atm.pagecontainer/pagecontainer.component';
+
+interface authPack {
+    email: string;
+    password: string;
+};
+
+const validateLogin = (pack:authPack) => {
+    const minimumSize = (pack.password.length >= 7);
+
+    if (!validateEmail(pack.email)){
+        Alert.alert("Email inválido " + pack.email)
+    }
+    else if (!validatePassword(pack.password)){
+        Alert.alert("Pelo menos 1 número e 1 caractere")
+    }
+    else if (!minimumSize){
+        Alert.alert("Senha muito curta (7 min.)")
+    }
+    else{
+        Alert.alert("Sucesso");
+    }
+}
+
+const validateEmail = (email: string) => {
+    const regexValidator = /.+[@].+\.com/;
+    const valid = regexValidator.test(email);
+    
+    return valid;
+}
+
+const validatePassword = (password: string) => {
+    const regexValidator = /((.*[A-Z].*)|(.*[a-z].*))(.*[0-9].*)/;
+    const valid = regexValidator.test(password);
+    
+    return valid;
+}
```

Comment:
Talvez fique melhor com esses métodos embaixo da page? Assim fica um modelo mais "historinha", o que é mais fácil pra quem vai ler e dar manutenção nessas pages.

Reply:
Realmente, mas eu tenho uma dúvida: uma boa prática seria jogar essas funções em um módulo separado talvez? como seria? Uma pasta /utils com essas coisas?

Reply:
É uma boa prática sim. Porque são blocos de código de fazem tarefas bem isoladas, e ainda podem ser reaproveitados em outras áreas do app, se for caso. Sobre a pasta `utils`, depende um pouco de como o projeto é organizado, mas dessa forma, pro caso do seu onboard, creio que já seja suficiente.